app/api/helpers/tasks.py

```diff
@@ -548,3 +559,212 @@ def delete_translations(self, zip_file_path):
+                                               new_invoice.amount,
+                                               app_name,
+                                               link,
+                                               new_invoice.event_id)
+
+
+@celery.task(base=RequestContextTask, name='send.event.fee.notification.followup')
+def send_event_fee_notification_followup():
+    from app.instance import current_app as app
+    with app.app_context():
+        incomplete_invoices = EventInvoice.query.filter(EventInvoice.status != 'paid').all()
+        for incomplete_invoice in incomplete_invoices:
+            if incomplete_invoice.amount > 0:
+                prev_month = monthdelta(incomplete_invoice.created_at, 1).strftime(
+                    "%b %Y")  # Displayed as Aug 2016
+                app_name = get_settings()['app_name']
+                frontend_url = get_settings()['frontend_url']
+                link = '{}/event-invoice/{}/review'.format(frontend_url,
+                                                           incomplete_invoice.identifier)
+                send_followup_email_for_monthly_fee_payment(incomplete_invoice.user.email,
+                                                            incomplete_invoice.event.name,
+                                                            prev_month,
+                                                            incomplete_invoice.amount,
+                                                            app_name,
+                                                            link)
+                send_followup_notif_monthly_fee_payment(incomplete_invoice.user,
+                                                        incomplete_invoice.event.name,
+                                                        prev_month,
+                                                        incomplete_invoice.amount,
+                                                        app_name,
+                                                        link,
+                                                        incomplete_invoice.event.id)
+
+
+@celery.task(base=RequestContextTask, name='expire.pending.tickets')
+def expire_pending_tickets():
+    from app.instance import current_app as app
+    with app.app_context():
+        db.session.query(Order).filter(Order.status == 'pending',
+                                       (Order.created_at + datetime.timedelta(minutes=30)) <= datetime.datetime.now()).\
+                                       update({'status': 'expired'})
+        db.session.commit()
+
+
+@celery.task(base=RequestContextTask, name='delete.ticket.holders.no.order.id')
+def delete_ticket_holders_no_order_id():
+    from app.instance import current_app as app
+    with app.app_context():
+        order_expiry_time = get_settings()['order_expiry_time']
+        TicketHolder.query.filter(TicketHolder.order_id == None, TicketHolder.deleted_at.is_(None),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [comparison to None should be 'if cond is None:'](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=4851924)